src/Sulu/Bundle/AdminBundle/Tests/Functional/Controller/AdminControllerTest.php

```diff
@@ -53,7 +59,8 @@ public function testGetConfig()
 
     public function testGetNotExistingMetdata()
     {
-        $client = $this->createAuthenticatedClient();
+        $client = $this->client;
```

Comment:
Instead of writing it like this I would prefer to use `$this->client` everywhere. But if it is too much work to change everywhere this works for me as well.